src/ActivitiesSDKAdapter.js

```diff
@@ -183,6 +201,33 @@ export default class ActivitiesSDKAdapter extends ActivitiesAdapter {
     );
   }
 
+  /**
+   * Encrypts the cards of an activity and returns a promise to the array of encrypted cards
+   *
+   * @param {object} activity  The activity that contains an array of adaptive cards
+   * @returns {Promise.<Array.<string>>} Promise that resolves to the array of encrypted cards
+   */
+  async encryptCards(activity) {
+    logger.debug('ACTIVITY', undefined, 'encryptCards()', ['called with', {activity}]);
+
+    let encryptedCards = [];
```

Comment:
no need for empty array initialization. In case of error, we don't want to return an empty array, we want to throw an error.

Reply:
actually, it might be easier to read without try/catch:

```js
const conversation = ...;

return Promise.all(activity.cards.map...
).catch(err => {
  logger.error(...);
  throw err;
});